### ISSUE TITLE
Provide multi-arch docker image (at least arm64)

### ISSUE THREAD

**Issue by timja** [Azure/Azurite#786]
### Which version of the Azurite was used?
mcr.microsoft.com/azure-storage/azurite:latest

### Where do you get Azurite? (npm, DockerHub, NuGet, Visual Studio Code Extension)
Dockerhub

### Steps to reproduce the issue?

Use an arm64 machine like an m1 mac

docker run -p 10000:10000 mcr.microsoft.com/azure-storage/azurite azurite-blob --blobHost 0.0.0.0

```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

### Have you found a mitigation/solution?

Docker has built-in support for emulating amd64, but it tends to not perform very well and crashed in more complicated applications

**Comment by XiaoningLiu** [comment]
Thanks for bringing this up! Will track here. The ideally way is to create an pipeline for auto build up images for different environments.

**Comment by lmsurpre** [comment]
https://hub.docker.com/_/microsoft-azure-storage-azurite still lists x86_64 as the only supported architecture, but I was somehow able to run it on my new MacBook Pro M1 with colima + docker... maybe its already addressed?

**Comment by XiaoningLiu** [comment]
> https://hub.docker.com/_/microsoft-azure-storage-azurite still lists x86_64 as the only supported architecture, but I was somehow able to run it on my new MacBook Pro M1 with colima + docker... maybe its already addressed?

We are handling https://github.com/Azure/Azurite/pull/1569 which adds multi arch support. The current published image is x86_64 only.

**Comment by blueww** [comment]
This is already supported. 
Both ARM64 and AMD64 docker image are released from 3.19.0.
